k8s/crds/kops_v1alpha2_cluster.yaml

```diff
@@ -2209,29 +2292,53 @@ spec:
                     logstashProbeTimer:
                       format: int32
                       type: integer
+                    monitorAggregation:
+                      type: string
                     nat46Range:
                       type: string
                     pprof:
                       type: boolean
+                    preallocateBPFMaps:
+                      type: boolean
                     prefilterDevice:
                       type: string
                     prometheusServeAddr:
                       type: string
                     restore:
                       type: boolean
+                    sidecarIstioProxyImage:
+                      type: string
                     singleClusterRoute:
                       type: boolean
                     socketPath:
                       type: string
                     stateDir:
                       type: string
+                    toFqdnsEnablePoller:
+                      type: boolean
                     tracePayloadlen:
                       format: int64
                       type: integer
                     tunnel:
                       type: string
                     version:
                       type: string
+                    waitBPFMount:
+                      type: boolean
+                  required:
+                  - enableipv6
+                  - enableipv4
+                  - monitorAggregation
+                  - bpfCTGlobalTCPMax
+                  - bpfCTGlobalAnyMax
+                  - preallocateBPFMaps
+                  - sidecarIstioProxyImage
```

Comment:
istio for cilium? correct me if I am wrong but they don't belong together I feel?

Reply:
Cilium can run alongside istio, docs are here: http://docs.cilium.io/en/v1.6/gettingstarted/istio